### DISCUSSION TITLE
Is it possible to trigger a Node method from the editor?

### DISCUSSION THREAD

**Discussion by ImGelu** [ueberdosis/tiptap#3107] [General Questions]
I'm currently developing an extension (Vue 3) that allows me to add some kind of inputs directly in a Tiptap editor. I've been trying for a while to auto-focus the input whenever the current cursor position matches the input's position, so it's not necessary to click it every time. So far, I managed to get the node using the `onTransaction` callback:

```        
        onTransaction({editor, transaction}) {
            if (transaction.curSelection.node && transaction.curSelection.node.type.name === "custom-input") {
                // TODO TRIGGER FOCUS ON INPUT
            }
        },
```

I've inspected the `transaction.curSelection.node` object, but couldn't find anything that would allow me to trigger something on my `Input` component/extension. I've tried adding an `update` attribute and manually changing the value whenever the cursor reaches a certain Node, and watching the changes via `watch()` in my component, but for some reason it only works once.

Is there any way to do that?

**Comment by bdbch** [comment]
Do you have some kind of Codesandbox I could play around with? Quite hard to grasp exactly without having something running in front of me! :)

**Reply by ImGelu** [reply]
Sure! Thanks :) https://codesandbox.io/s/tiptap-firefox-inputs-forked-l1uvk0

The idea is that whenever my keyboard cursor reaches the component, the input is automatically focused.
